targets/chromium

```diff
@@ -3,9 +3,9 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 if [ "${TARGETNOINSTALL:-c}" = 'c' ] &&
-        [ "$DISTRO" = 'debian' -o "$DISTRO" = 'kali' ] &&
+        [ "$DISTRO" = 'debian' -o "$DISTRO" = 'kali' -o "$DISTRO" = 'opensuse' ] &&
         [ "${ARCH#arm}" != "$ARCH" ]; then
-    error 99 "chromium target is not supported on Debian/ARM."
+    error 99 "chromium target is not supported on "$DISTRO"/"$ARCH"."
```

Comment:
this quoting isn't quite what you want

Reply:
It's only available for x86, not for ARM.

Reply:
would something like this be acceptable for you:
    error 99 "chromium target is not supported on "$DISTRO"/ARM."

Reply:
you don't need the quotes around $DISTRO, since the entire string is quoted

Reply:
and I'd prefer "chromium target is not supported on $DISTRO on ARM"

Reply:
sure, no problem